Sources/Plasma/FeatureLib/pfMetalPipeline/ShaderSrc/PlateShaders.metal

```diff
@@ -83,12 +82,11 @@ vertex ColorInOut plateVertexShader(PlateVertex in [[stage_in]],
     out.position.y *= -1.f;
     out.texCoord = (float4(in.texCoord, 1.0) * uniforms.uvTransforms[0].transform).xyz;
     out.texCoord.y = 1.f - out.texCoord.y;
-    out.normal = float4(0.f, 0.f, 1.f, 0.f);
 
     return out;
 }
 
-fragment float4 fragmentShader(ColorInOut in [[stage_in]],
+fragment half4 fragmentShader(ColorInOut in [[stage_in]],
```

Comment:
Looks like the subsequent lines need their indentation adjusted.

Reply:
I've amended the commit.